databricks_cli/pipelines/cli.py

```diff
@@ -95,14 +95,19 @@ def deploy_cli(api_client, spec_arg, spec, allow_duplicate_names, no_update_spec
         click.echo("Successfully created pipeline: {}".format(
             _get_pipeline_url(api_client, new_pipeline_id)))
 
-        if not no_update_spec:
-            spec_obj['id'] = new_pipeline_id
-            _write_spec(src, spec_obj)
-            click.echo("Updated spec at {} with ID {}".format(src, new_pipeline_id))
-        else:
-            click.echo("Pipeline has been assigned ID {}".format(new_pipeline_id))
+        click.echo("Pipeline has been assigned ID {}".format(new_pipeline_id))
+        click.echo(new_pipeline_id, err=True)
```

Comment:
What's this second line for?

Reply:
Also, can we swap that "assigned ID" line to be before the link?

Reply:
> What's this second line for?

This was the thing where we log only the pipeline_id to STDERR so people can pipe it more easily. 

> Also, can we swap that "assigned ID" line to be before the link?

What do you mean by this?

Reply:
Oh, just moving the new logging you added to be before the previous one.

Reply:
Done.